autoload/previm.vim

```diff
@@ -96,8 +97,25 @@ endfunction
 
 let s:base_dir = fnamemodify(expand('<sfile>:p:h') . '/../preview', ':p')
 
+function! s:fix_preview_base_dir() abort
+  if exists('g:previm_custom_preview_base_dir')
+    let s:preview_base_dir = expand(g:previm_custom_preview_base_dir)
+    if !isdirectory(s:preview_base_dir . '_/js/previm.js')
```

Comment:
filereadable?

Reply:
or something using preview_base_dir (at least, it should NOT be preview.js)

Reply:
> filereadable?

yes. I'll fix it